openedx/core/djangoapps/user_api/accounts/tests/test_api.py

```diff
@@ -302,7 +302,6 @@ class AccountCreationActivationAndPasswordChangeTest(TestCase):
         '@domain.com',
         'test@no_extension',
         u'fŕáńḱ@example.com',
-        u'frank@éxáḿṕĺé.ćőḿ',
```

Comment:
Now this is a valid email according to [validate_email](https://docs.djangoproject.com/en/1.8/ref/validators/#django.core.validators.validate_email)